exporters/export_formatter/avro_export_formatter.py

```diff
@@ -0,0 +1,68 @@
+import json
+import six
+from fastavro.writer import Writer
+import io
+
+from exporters.exceptions import ConfigurationError
+from exporters.export_formatter.base_export_formatter import BaseExportFormatter
+
+
+class AvroExportFormatter(BaseExportFormatter):
+    """
+    This export formatter provides a way of exporting items in AVRO format.
+    follows the avro schema 1.8 https://avro.apache.org/docs/1.8.1/spec.html
+    We write the items on a 1 per block way.
+    This allows for streaming and keeps the files consistency even if the process
+    ends unexpectedly
+
+        - schema(string)
+            json-encoded schema for the data to use in avro formatter
+
+        - schema_path(string)
+            path to the json schema for the data to use in avro formatter.
+            Useful for large schemas if readibility of the job json file is important
+    """
+
+    supported_options = {
+        'schema_path': {'type': six.string_types, 'default': None},
+        'schema': {'type': six.string_types, 'default': None},
+    }
+    file_extension = 'avro'
+    item_separator = ''
+
+    def __init__(self, *args, **kwargs):
+        super(AvroExportFormatter, self).__init__(*args, **kwargs)
+        self.schema = self._get_schema()
+        self._setup_writer()
+
+    def _get_schema(self):
+        if self.read_option('schema_path'):
+            with open(self.read_option('schema_path')) as fname:
+                return json.load(fname)
+        elif self.read_option('schema'):
+            return json.loads(self.read_option('schema'))
+        else:
+            raise ConfigurationError('Avro formatter requires at least one of: schema or schema_path')
```

Comment:
This is the line that is failing the flake8 check:

```
./exporters/export_formatter/avro_export_formatter.py:45:101: E501 line too long (102 > 100 characters)
```

You can just break it like:

```
raise ConfigurationError(
    'Avro formatter requires at least one of: schema or schema_path')
```

Reply:
Yeah that makes sense. Wasn't keen on reformatting the whole message for 2 characters but this way  works fine.